AdminCore.Services/EventService.cs

```diff
@@ -584,5 +583,16 @@ private static int AutoApproveEventsNotNeedingAdminApproval(EventTypes eventType
 
       return eventStatusId;
     }
+
+    private void CheckEventTypeAdminLevel(EventTypes eventTypes, int employeeId)
+    {
+      int eventTypeId = (int)eventTypes;
+      var levelRequired = DatabaseContext.EventTypeRepository.GetSingle(x => x.EventTypeId == eventTypeId).EmployeeRoleId;
```

Comment:
Very small change, but rather than pulling back the whole event or employee, you can write:

DatabaseContext.EmployeeRepository.GetAsQueryable(x => x.EmployeeId == employeeId).Select(x => x.EmployeeRoleId).FirstOrDefault();